R/Rules-methods.R

```diff
@@ -1481,6 +1481,61 @@ setMethod("stopTrial",
           })
 
 
+## --------------------------------------------------
+## Stopping based on precision of MTD calculated as CV(MTD)
+## --------------------------------------------------
+
+##' @describeIn stopTrial Stopping rule based precision of the MTD estimation
+##' The trial is stopped, when the MTD can be estimated with sufficient precision.
+##' The criteria is based on the robust CV calculated from the posterior distribution.
+##' The robust coefficient of variation is defined as MAD(MTD)/median(MTD)
+
+setMethod("stopTrial",
+  signature =
+    signature(
+      stopping = "StoppingMTDCV",
+      dose = "numeric",
+      samples = "Samples",
+      model = "Model",
+      data = "ANY"
+    ),
+  def =
+    function(stopping, dose, samples, model, data, ...) {
+      # First, generate the MTD samples.
+
+      # add prior data and samples to the
+      # function environment so that they
+      # can be used.
+      MTDSamples <- dose(
```

Comment:
```suggestion
      mtd_samples <- dose(
```
while we keep camelCase for functions and classes, internal variables good to use snake_case